pytorch/inference/docker/2.2/py3/cu118/Dockerfile.ec2.gpu.os_scan_allowlist.json

```diff
@@ -1,7 +1,7 @@
 {
   "linux": [
     {
-      "description": "\n It was discovered that a race condition existed in the Rose X.25\n protocol implementation in the Linux kernel, leading to a use-after-\n free vulnerability. A local attacker could use this to cause a denial\n of service (system crash) or possibly execute arbitrary code.",
+      "description": "\n It was discovered that a race condition existed in the Rose X.25 protocol\n implementation in the Linux kernel, leading to a use-after- free\n vulnerability. A local attacker could use this to cause a denial of service\n (system crash) or possibly execute arbitrary code.",
```

Comment:
We are not release EC2, right?

Reply:
No, I'm just updating the CVEs for all of them for convenience